pertpy/plot/_augur.py

```diff
@@ -95,12 +95,10 @@ def important_features(
 
         ag = Augur("random_forest_classifier")
 
-        return ag.plot_important_features(data=data, key=key, top_n=top_n, ax=ax, return_figure=return_figure)
+        return ag.plot_important_features(data=data, key=key, top_n=top_n, ax=ax)
 
     @staticmethod
-    def lollipop(
-        data: dict[str, Any], key: str = "augurpy_results", ax: Axes = None, return_figure: bool = False
-    ) -> Figure | Axes:
+    def lollipop(data: dict[str, Any], key: str = "augurpy_results", ax: Axes = None) -> Figure | Axes:
```

Comment:
`plot_dp_scatter` for example returns `Figure | Axes | None`. Why does this one not return `| None`?